src/appleseed/renderer/modeling/aov/depthaov.cpp

```diff
@@ -80,8 +81,7 @@ namespace
 
         virtual void flush(ShadingResult& result) override
         {
-            result.m_aovs[m_index].m_color.set(m_depth);
-            result.m_aovs[m_index].m_alpha.set(1.0f);
+            result.m_aovs[m_index] = Color4f(m_depth, m_depth, m_depth, 1.0f);
```

Comment:
Yeah, it bothers me a little sometimes too.

`m_avos[m_index]` is a `foundation::Color4f`, which has a `set()` method to set all components to the same value, but if you want to set all components individually, right now there's no other choice than making a copy. I'm pretty sure the compiler will elide this copy, but having a `set(r, g, b, a)` method would be another choice.